content/en/docs/contributing/localization.md

```diff
@@ -16,24 +16,48 @@ as can be seen from the languages dropdown menu in the top nav.
 When translating website pages from English, we recommend that you follow the
 guidance offered in this section.
 
-### Summary / TL;DR {#summary}
+### Summary
 
-Do translate:
+#### ✅ Do {#do}
 
-- [Front matter][] fields `title`, `linkTitle`, and `description`
-- Page content, including the text fields in Mermaid [diagrams](#images)
+<div class="border-start border-success bg-success-subtle">
 
-Do **NOT**:
+- **Translate**:
+  - Page content, including:
+    - Mermaid [diagram](#images) text fields
+    - Code comments from code excerpts
```

Comment:
I'm not 100% sure about code comments being in the "do" list, because I don't want to force all locals to edit the code. Thoughts @theletterf @tiffany76?

Reply:
Here Be Lions. :) I'll leave that to you.

Reply:
Let's leave it for now and revisit as needed.